src/components/NewRequest.jsx

```diff
@@ -1,15 +1,22 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { Form, Input } from "semantic-ui-react";
 import axios from "axios";
 import createHeaders from "../modules/headers";
-import { useSelector } from "react-redux";
+import { useSelector, useDispatch } from "react-redux";
 import { Redirect } from "react-router-dom";
+import getKarma from "../modules/getKarma";
 
 const NewRequest = () => {
   const [message, setMessage] = useState("");
+  const dispatch = useDispatch();
   const authenticated = useSelector(
     (state) => state.authentication.authenticated
   );
+  const karmaPoints = useSelector((state) => state.karma.karma);
```

Comment:
is it needed to get the karma if you have it in the useEffect?

Reply:
In useEffect, it is updating the redux state, we grab the state from redux with useSelector